op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -501,6 +501,18 @@ func runDBCheck(opts dbCheckOptions) (err error) {
 
 	log.Info("DB Continuity Check Started", "dbPath", opts.dbPath)
 
+	// We want to open the db in readonly mode to take advantage of concurrency, but opening in readonly
+	// mode will fail if the db is missing some files suffixed by ".meta". Our legacy celo db does not have
+	// ".meta" files, so opening in readonly mode will fail. To work around this, we open the db in readwrite
+	// mode and then close it so that the ".meta" files are created. We then reopen the db in readonly mode.
+	db, err := openDB(opts.dbPath, false)
+	if err != nil {
+		return fmt.Errorf("failed to open db in readwrite mode: %w", err)
+	}
+	if err = db.Close(); err != nil {
+		return fmt.Errorf("failed to close db in readwrite mode: %w", err)
+	}
+
 	ancientDB, err := NewChainFreezer(filepath.Join(opts.dbPath, "ancient"), "", true)
```

Comment:
With this change can we change how we open the db for the pre and full migrations, which are currently not using readonly.

Reply:
In theory, but if we ever run those scripts without the continuity script before them for whatever reason they'll have this problem. It would be creating a hard dependency between the scripts. What do you think about that tradeoff?